README.md

```diff
@@ -1 +1,2 @@
-# EmbeddingImagesAndVideo
\ No newline at end of file
+# EmbeddingImagesAndVideo
+## This project consists of building an HTML document that matches the appearance of youtube video player page.
```

Comment:
`AWESOME`

Nice job creating a `README.md` file, this gives the project more description of what it does and how it does it.

-----

`REQUIRED`

> Remember about descriptive README file.`

- Add the assignment link, the project specification, screenshots and other pieces of information that would describe the project.